com.unity.render-pipelines.high-definition/Editor/Lighting/LightUnit/LightUnitSlider.cs

```diff
@@ -503,8 +503,8 @@ void PrepareExponentialConstraints(float lo, float mi, float hi)
             // If for any reason the constraints are changed, then the function must be refit and the new coefficients found.
             // Note that we can't re-use the original PowerSlider instead due to how it forces a text field, which we don't want in this case.
             m_ExponentialConstraints.x = -3935.53965427f;
-            m_ExponentialConstraints.y = 5435.53965427f;
-            m_ExponentialConstraints.z = 1.48240556f;
+            m_ExponentialConstraints.y =  5435.53965427f;
+            m_ExponentialConstraints.z =     1.48240556f;
```

Comment:
I assume these changes won't survive the new C# formatting.

Reply:
yeah, it was a missing format